sci-electronics/fritzing/fritzing-0.9.4.ebuild

```
@@ -0,0 +1,94 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit qmake-utils xdg
+
+MY_PV="CD-498"
+PARTS_P="${PN}-parts-${PV}"
+PARTS_COMMIT="e79a69765026f3fda8aab1b3e7a4952c28047a62"
+DESCRIPTION="Electronic Design Automation"
+HOMEPAGE="https://fritzing.org/"
+SRC_URI="https://github.com/fritzing/fritzing-app/archive/${MY_PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/fritzing/fritzing-parts/archive/${PARTS_COMMIT}.zip -> ${PARTS_P}.zip"
```

Please make some newlines here to make this mess more readable.
```suggestion
MY_PV="CD-498"
PARTS_P="${PN}-parts-${PV}"
PARTS_COMMIT="e79a69765026f3fda8aab1b3e7a4952c28047a62"

DESCRIPTION="Electronic Design Automation"
HOMEPAGE="https://fritzing.org/"
SRC_URI="https://github.com/fritzing/fritzing-app/archive/${MY_PV}.tar.gz -> ${P}.tar.gz
	https://github.com/fritzing/fritzing-parts/archive/${PARTS_COMMIT}.zip -> ${PARTS_P}.zip"
```